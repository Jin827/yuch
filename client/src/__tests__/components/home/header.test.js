import React from 'react';
import { render } from '../../setupTests';
import Header from '../../../components/home/header';

test('<Header /> renders correctly', () => {
  const { container } = render(<Header />);
  expect(container.firstChild).toMatchSnapshot();
});
