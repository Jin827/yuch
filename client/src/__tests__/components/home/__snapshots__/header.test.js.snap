// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header /> renders correctly 1`] = `
<header>
  <div
    class="flex justify-between header--center"
  >
    <button
      aria-label="arrowLeft"
      style="background-color: transparent; padding-top: 0px; padding-bottom: 0px;"
      type="button"
    >
      <svg
        class="svg-icon icon-arrowLeft || ''"
        height="27"
        viewBox="0 0 24 24"
        width="27"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M0 0h24v24H0V0z"
          fill="none"
          opacity=".87"
        />
        <path
          d="M17.51 3.87L15.73 2.1 5.84 12l9.9 9.9 1.77-1.77L9.38 12l8.13-8.13z"
          fill="#ffffff"
        />
      </svg>
    </button>
    <button
      aria-label="arrowRight"
      style="background-color: transparent; padding-top: 0px; padding-bottom: 0px;"
      type="button"
    >
      <svg
        class="svg-icon icon-arrowRight || ''"
        height="27"
        viewBox="0 0 24 24"
        width="27"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M24 24H0V0h24v24z"
          fill="none"
          opacity=".87"
        />
        <path
          d="M6.49 20.13l1.77 1.77 9.9-9.9-9.9-9.9-1.77 1.77L14.62 12l-8.13 8.13z"
          fill="#ffffff"
        />
      </svg>
    </button>
  </div>
  <div
    class="header-text--box"
  >
    <h1>
      <div
        class="header--no-msg f-en"
      >
        NO MSG!
      </div>
      <br />
      오늘도 열심히 일한 당신에게 
      <br />
      당신만을 위한
       
      <span
        class="header--text--point-color"
      >
        착한 가격의 집밥을
      </span>
      선물하세요.
    </h1>
  </div>
</header>
`;
